types/sceditor/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please remove this and make this file just `{ "extends": "dtslint/dt.json" }` -- new packages shouldn't need lint rule disables.